### DISCUSSION TITLE
Donate button missing

### DISCUSSION THREAD

**Discussion by boss1819** [apernet/hysteria#796] [General]
on the website there is a donate page, but no link or real info how to donate. only an svg of the nowpayments. 
then at the end of the page below (or any other page in the website), there is a very small non visible link for donation... suggestion to add this as well to the donation page itself.

**Comment by tobyxdd** [comment]
Oof, thanks for the tip. It was supposed to be clickable, but the introduction of the lightbox plugin broke it. It's fixed now.
